src/elements/option/option/option.ts

```diff
@@ -273,13 +277,13 @@ export class SbbOptionElement extends SbbDisabledMixin(SbbIconNameMixin(LitEleme
     }
   }
 
-  private _setupHighlightHandler(event: Event): void {
+  private _handleHighlightState(): void {
     if (!this._isAutocomplete) {
       this._updateDisableHighlight(true);
       return;
     }
 
-    const slotNodes = (event.target as HTMLSlotElement).assignedNodes();
+    const slotNodes = Array.from(this.childNodes ?? []);
```

Comment:
What's the sense of this line change?

Reply:
The previous logic depended on the event, but with the change in calling it in connectedCallback, I have no event in that case.
I just realized there is a small distinction. I have to exclude elements that are assigned to the image slot.